tools/cli/wskaction.py

```diff
@@ -188,11 +188,14 @@ def getExec(self, args, props):
                 exe['kind'] = 'java'
                 exe['jar'] = base64.b64encode(contents)
                 exe['main'] = self.findMainClass(args.artifact)
+            elif args.kind in ['nodejs']:
+                exe['kind'] = 'nodejs'
```

Comment:
We currently have NodeJS 0.12.x and NodeJS 6.x. A `kind` of "nodejs" will always mean Node JS 0.12.x so that existing Actions do not need to have their `kind` values migrated.  Henceforth, NodeJS runtime `kind` values will be explicit about the actual version they represent - `nodejs:{version}`